subprojects/docs/src/docs/userguide/getting_started.adoc

```diff
@@ -19,7 +19,7 @@ Everyone has to start somewhere and if you're new to Gradle, this is where to be
 
 == Before you start
 
-In order to use Gradle effectively, you need to know what it is and understand some of its fundamental concepts. So before you start using Gradle in earnest, we highly recommend you read <<what_is_gradle#what_is_gradle,What is Gradle?>>.
+In order to use Gradle effectively, you need to know what it is and understand some of its fundamental concepts. So before you start using Gradle in earnest, we highly recommend you read <<userguide#what_is_gradle,What is Gradle?>>.
```

Comment:
```suggestion
In order to use Gradle effectively, you need to know what it is and understand some of its fundamental concepts. So before you start using Gradle in earnest, we highly recommend you read <<userguide#what_is_gradle,What is Gradle?>>
```

Reply:
Hey @tresat, thank you. 

Notes:
- The Getting Started page and Quick Start section will be updated entirely in https://github.com/gradle/gradle/pull/24913.
- When the Getting Started / Quick Start section is updated, I will be able to add the links in the new home page (I can't do it now or I will get dead link errors in the build - I will do it in PR [24913](https://github.com/gradle/gradle/pull/24913) instead).

Reply:
Sounds good!